app/forms/provider_onboarding_form.rb

```diff
@@ -28,4 +28,16 @@ def provider_already_onboarded?
       errors.add(:code, :already_added, provider_name: provider.name)
     end
   end
+
+  def code_presence
+    errors.add(:code, code_error_message) if code.blank?
```

Comment:
I'm not positive this is where the issue is, but the href in the error summary does not take you to the input field. The error summary is looking for a field with id "provider-code-field-error"
<img width="798" alt="image" src="https://github.com/DFE-Digital/itt-mentor-services/assets/44073106/fb77f1de-1884-4068-b4e7-bb1b9fbfa239">